CONTRIBUTING.md

```diff
@@ -107,7 +102,9 @@ If you're only modifying the helm chart we recomment you enter nix-shell from th
 >
 > That's totally fine, just make sure you have `helm-docs` installed and run the docs script
 > ```bash
-> [@dev:~/git/mayastor/extensions]$ ./scripts/helm/> generate-readme.sh
+> [@dev:~/git/mayastor/extensions]$ ./scripts/helm/generate-readme.sh
+> INFO[2023-08-22T08:55:06+01:00] Found Chart directories [chart]
+> INFO[2023-08-22T08:55:06+01:00] Generating README Documentation for chart chart
```

Comment:
This probably needs to be fixed in the scripts. But just pointing it out here. 

`for chart chart` --> `for chart`